launcher_scripts/nemo_launcher/core/k8s_templates/rlhf_ppo/values.yaml

```diff
@@ -0,0 +1,39 @@
+image:
+  trainingImage: cfg.container
+  pullPolicy: IfNotPresent
+
+  # Insert the name of your container registry pull secret #
+  pullSecret: nvcr.io
+
+trainingConfig:
+  # Specify the amount of shared memory to attach to the Pods #
+  shmSize: 512Gi
+
+  # Insert the address for the NFS server if using NFS for model storage #
+  NFSServer: <Insert NFS server address>
+
+  # Insert the path to save data on the NFS server #
+  NFSPath: <Insert NFS server path>
+
+  # Specify the k8s resource name for IB devices #
+  ibResourceName: nvidia.com/hostdev
+
+  # Specity the number of IB devices to include in pods #
+  ibCount: "0"
+
+  # Specify the WandB API key if using WandB for logging #
+  wandbKey: "nil"
+
+  # Insert the dnsPolicy #
+  dnsPolicy: "nil"
+
+  # Specify the namespace
+  namespace: "default"
```

Comment:
Yes, agreed.

Reply:
Effectively you'd remove this line here and then plumb the namespace optional here instead: https://github.com/NVIDIA/NeMo-Megatron-Launcher/blob/master/launcher_scripts/nemo_launcher/core/launchers.py#L552

Reply:
Yes, I'm thinking how to plumb the namespace to here. Because the typical usage is to run '`python main.py`', there is no way for customer to input a `namespace`.

Reply:
helm can render {{.Release.namespace}}, that's a good way. Thanks.

Reply:
Put the namespace into cluster/k8s.yaml. Test with different settings:
1. Set namespace: "default", and check results in rlhf_2b_actor_2b_critic_submission.sh:
 `#!/bin/bash
helm template --timeout=15m --wait  --namespace default rlhf-2b-actor-2b-critic /nemo/eliu/NeMo-Megatron-Launcher/launcher_scripts/results/rlhf_2b_actor_2b_critic/k8s_template`
2. Set namespace: "", and check results in rlhf_2b_actor_2b_critic_submission.sh:
 `#!/bin/bash
helm template --timeout=15m --wait rlhf-2b-actor-2b-critic /nemo/eliu/NeMo-Megatron-Launcher/launcher_scripts/results/rlhf_2b_actor_2b_critic/k8s_template`